tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -515,6 +529,19 @@ dhcp_relay/test_dhcp_relay.py::test_dhcp_relay_unicast_mac:
       - "'dualtor' in topo_name and release in ['201811', '201911']"
       - "platform in ['x86_64-8111_32eh_o-r0']"
 
+dhcp_relay/test_dhcp_relay.py::test_dhcp_relay_with_source_port_ip_in_relay_enabled:
```

Comment:
![image](https://github.com/user-attachments/assets/0fa24cb8-880a-4bb4-bdd0-cb861dd241ea)
The skip condition should be dualtor and old release